WooCommerce/UITestsFoundation/Screens/Login/PasswordScreen.swift

```diff
@@ -21,10 +21,8 @@ public final class PasswordScreen: BaseScreen {
         super.init(element: passwordTextField)
     }
 
-    public func proceedWith(password: String) -> LoginEpilogueScreen {
+    public func proceedWith(password: String) {
```

Comment:
This can now end up in `LoginEpilogueScreen` or `MyStoreScreen` depending on the login flow so I removed the return value. Not sure if this is the best way to do it though, open to suggestions.